### PR TITLE
Timeline component

### LINKED ISSUE #13: Add a simple vertical timeline component
### Feature request-
Add a simple vertical timeline component with white background.

https://user-images.githubusercontent.com/54665036/120939014-59df5480-c733-11eb-8e1e-825b01b9126b.mp4

<hr/>

**Note:**

-   If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
-   If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
- It is adviced to take up/ work on only 2 issues at the same time.
-   Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.

### PR DESCRIPTION
**Pull request by SamikshaSankar**
## Related issue?
closes: #13 

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

-   [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
-   [] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/Mirage-UI/pulls) for the same update/change?
-   [] Have you made corresponding changes to the documentation?
-   [] Have you tested the code before submission?
-   [] Did you lint and format your code before making the Pull Request?
-   [] Have you made the GIF for the component you added?

## GIF you created:
Add here the GIF you created.

### CODE REVIEW

File: HTML+CSS+JS/README.md [hunk 1]
```diff
@@ -73,4 +73,5 @@
 
     -   [Vertical Timeline](Timelines/VerticalTimeline)
     -   [Reading Progress Bar](ReadingProgressBar)
+    -   [Timeline component](TimeLine component)
```
Comment (theBeginner86) [inline HTML+CSS+JS/README.md]: Here, is the error I am talking about.
You can even use dashes '-' to separate the words.

**Review by theBeginner86 — CHANGES_REQUESTED** [review]
Good work 👍
Rest everything is working fine.

If you could change the name of your folder from `TimeLine component` ---> `TimeLineComponent` or something like that which doesn't have space between the words then that would be fine.
I am saying this because it is in accordance with good practice and would also remove the linking error which is coming due to it in the HTML+CSS+JS/README.md file.

#### Just one more thing, from next time please make use of the PR template and don't alter it as it helps us to understand your changes, and please do add a screenshot/demo video of the changes so that we can easily review it. 🙂

File: HTML+CSS+JS/README.md (hunk 1 above)
Comment (theBeginner86) [reply HTML+CSS+JS/README.md]: ✨✨**Tip**

For example, you make the new name `TimeLineComponent`
Then in the  HTML+CSS+JS/README.md file, you just need to replace the above-selected line with:
`- [TimeLineComponent](Timelines/TimeLineComponent)`
Though take care of indentation and also test it before pushing.  😁🙂

Comment (SamikshaSankar) [reply HTML+CSS+JS/README.md]: Sorry for that. Thanks for help...next time I will do that things

**Review by Pranav016 — APPROVED** [review]
Next time try keeping commits to the minimum 👍

**Pull request merged**